recept.cpp

```diff
@@ -103,22 +36,22 @@ void filter(char* buf) {
 	// type == 3 && code == 26 -> value == tilt x
 	// type == 3 && code == 27 -> value == tilt y
 
+
 	// pen/eraser in
 	if (type == 1 && ((code == 320 && value == 1) || (code == 321 && value == 1))) {
-
-		ring_x.clear();
-		ring_y.clear();
+		ema_x = value; // Reset EMA when pen/eraser is touching
```

Comment:
May need some initialization/clearing, because sometimes after wakeup a line (maybe from last pen position before sleep?) is drawn to the new pen point.